notes/returned-mail-received.vbs

```diff
@@ -307,65 +333,73 @@ IF mailing_address_confirmed = "YES" or residential_address_confirmed = "YES" TH
         		cancel_without_confirmation
         		If isdate(date_requested) = FALSE THEN  err_msg = err_msg & vbnewline & "Please enter the date verifications were requested."
 				IF Cdate(date_requested) > cdate(date) = TRUE THEN  err_msg = err_msg & vbnewline & "You must enter an actual date that is not in the future."
-        		IF ECF_reveiwed = "Select One:" THEN  err_msg = err_msg & vbnewline & "Please review ECF to ensure the requested verifications are not on file."
-        		IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbCr & err_msg & vbCr & vbCr & "Please resolve for the script to continue."
-        	LOOP UNTIL err_msg = ""
+        		IF ECF_reveiwed = "Select:" THEN  err_msg = err_msg & vbnewline & "Please review ECF to ensure the requested verifications are not on file."
+	    	LOOP UNTIL err_msg = ""
         	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to      password back into MAXIS
         LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
 
         'per POLI/TEMP this only pertains to active cash and snap '
-        IF case_active = TRUE THEN
+        IF snap_or_cash_case = TRUE THEN
```

Comment:
There are no restrictions here to ensure 10 days have passed.

Reply:
added a verification_received_date variable to ensure 10 days has been allowed